cmd/psqldef/psqldef.go

```diff
@@ -104,5 +106,8 @@ func main() {
 	}
 	defer database.Close()
 
-	sqldef.Run(schema.GeneratorModePostgres, database, options)
+	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	defer cancel()
```

Comment:
Is it OK to propagate context cancel  by syscall?
I would like to ask the maintainer's intention regarding the use case of canceling the migration.